packages/reason-relay/src/ReasonRelay.re

```diff
@@ -962,7 +961,7 @@ module type MakeUsePaginationFragmentConfig = {
 };
 
 type paginationLoadMoreOptions = {
-  onComplete: option(option(Js.Exn.t) => unit),
+  onComplete: option(Js.nullable(Js.Exn.t) => unit),
```

Comment:
Previously I've opted to autoconvert these to options for the user. I'm thinking we should do that too here, even if it's a bit more cumbersome.

Reply:
...maybe that's already done and I'm just looking at the wrong thing?

Reply:
actually, this is the type of the bindings, I'm doing the autoconversion later with `nullableToOptionalExnHandler`for user-facing APIs, let me know if you've found APIs where that's not the case. Since I haven't touched the interface file I think it's alright. Thank you again .rei files :)

Reply:
Yeah you're absolutely right, great stuff!